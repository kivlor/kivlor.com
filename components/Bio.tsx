/** @jsxImportSource preact */

export default function Bio() {
  return (
    <>
      <p class="mb-2 font-semibold text-center text-gray-900 text-2xl md:text-3xl">
        Andrew Gallagher, <em class="font-normal">aka</em> Kivlor
      </p>

      <p class="mb-2 font-light text-center text-gray-600 text-xl md:text-2xl">
        Maker, Coder, <em class="line-through not-italic">Tourist</em>, Geek.
      </p>
    </>
  );
}
