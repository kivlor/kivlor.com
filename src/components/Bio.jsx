import * as React from 'react';

export const Bio = () => (
  <section className="bio">
    <p>Andrew Gallagher, <em>aka</em> Kivlor.</p>
    <p>Maker, Coder, Tourist, Geek.</p>
  </section>
);
