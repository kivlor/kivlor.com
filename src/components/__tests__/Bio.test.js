import React from 'react';
import renderer from 'react-test-renderer';

import { Bio } from '../Bio';

describe('Bio', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Bio />);

    expect(component).toBeTruthy();
  });

  it('matches snapshot', () => {
    const component = renderer.create(<Bio />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
