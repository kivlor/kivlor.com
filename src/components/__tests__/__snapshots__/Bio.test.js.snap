// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio matches snapshot 1`] = `
<section
  className="bio"
>
  <p>
    Andrew Gallagher, 
    <em>
      aka
    </em>
     Kivlor.
  </p>
  <p>
    Maker, Coder, Tourist, Geek.
  </p>
</section>
`;
